README.md

```diff
@@ -1,4 +1,31 @@
 # Kubernetes Metrics Server
 
-more details TBD
+## User documentation
 
+User documentation can be found in
+[the official Kubernetes documentation](https://kubernetes.io/docs/tasks/debug-application-cluster/core-metrics-pipeline/).
+
+## Design
+
+The project was designed in two phases:
+
+- [Metrics API](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/instrumentation/resource-metrics-api.md)
+- [Metrics Server](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/instrumentation/metrics-server.md)
+
+For the broader view of monitoring in Kubernetes take a look into
+[Monitoring architecture](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/instrumentation/monitoring_architecture.md)
```

Comment:
I think monitoring architecture should be moved above. Smth like this:

```
You can find more information about the high-level design in the [Monitoring architecture][...]. The two most important parts are:

- [Metrics API][...]
- [Metrics Server][...]
```

Reply:
I disagree. You should be able to understand the design of metrics server without being familiar with the whole architecture. It's rather a "further reading" category here.

Reply:
OK, that's fine with me also

Could you please rephrase `the project was designed in two phases`? I don't think anyone cares in which order it was designed, the important part is that the whole thing consist of two separate pieces